src/pretix/settings.py

```diff
@@ -209,22 +214,28 @@
 
 HAS_REDIS = config.has_option('redis', 'location')
 if HAS_REDIS:
+    OPTIONS = {
```

Comment:
Please rename it to `redis_options`. We don't expose it as application settings, no reason to make it all-uppercase.